strategy-offline-fallback/README.md

```diff
@@ -0,0 +1,17 @@
+# Offline fallback
+The recipe provides a service worker serving a default offline fallback in case
+of missing resource.
+
+## Difficulty
+Intermediate
+
+## Use Case
+You want to fail in a controlled way while serving resources providing content
```

Comment:
I think I'd say something like "you want to make sure the users always receive some content, even if the network is not available"